src/renderer/Components/RecevingPackage/NumberInput/index.jsx

```diff
@@ -1,20 +1,20 @@
 import PropTypes from "prop-types";
 
-function NumberInput({ validateNumber, handleMoveNextInput }) {
+function NumberInput({ onKeyDownFunc, OnChangeFunc }) {
   return (
     <input
       maxLength="1"
       className="mx-3 h-16 w-16 transform rounded-lg bg-gray-200 text-center text-5xl shadow-md transition duration-200 hover:scale-110 focus:outline-none"
       type="text"
-      onKeyDown={(event) => validateNumber(event)}
-      onChange={(event) => handleMoveNextInput(event)}
+      onKeyDown={(event) => onKeyDownFunc(event)}
+      onChange={(event) => OnChangeFunc(event)}
```

Comment:
```suggestion
      onChange={(event) => OnChangeFunc(event)}
       onChange={(event) => onChangeFunc(event)}
```

이벤트 함수명이 PascalCase 로 작성된거 같습니다.
camelCase 로 수정부탁드리겠습니다.😄

Reply:
확인한 뒤 처리하였습니다!